### ISSUE TITLE
Can't install new packages

### ISSUE THREAD

**Issue by dnicloud** [wbond/package_control#1185]
I realized like a week ago that I cannot install any new packages via Package Control. I tried everything even reinstalling sublime, while cleaning all files ~/Library/Application Support/ everything. Still the same problem. Any ideas?

`ignored packages updated to: ["Emmet", "Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
Package Control: Error downloading package. HTTP exception InvalidCertificateException (Host codeload.github.com returned an invalid certificate ([SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:548))) downloading https://codeload.github.com/sergeche/emmet-sublime/zip/master.
error: Package Control

Unable to download Emmet. Please view the console for more details.
ignored packages updated to: ["Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings`

**Comment by wbond** [comment]
Other users have reported the issue, but no one who has had the issue has taken the time to figure it out, and most users can't replicate the issue. So users who are experiencing the bug are sort of stuck until someone figures it out.

Duplicate of #1163.

**Comment by wgregp** [comment]
(Duplicating my comment from #1163) Same issue, but I managed to solve it. It appears that the default `Package Control.merged-ca-bundle` file (as installed today, 9 Nov 2016) does not include the `DigiCert High Assurance EV Root CA` certificate, which is what GitHub uses. I exported that certificate from Keychain Access as a `.pem` file, pasted the contents into `Package Control.user-ca-bundle`, restarted Sublime Text, and everything now seems to work.

**Comment by wbond** [comment]
@wgregp Interesting. I’ve got in mind to add some debug info to the export process, so hopefully that can help in identifying why the cert wasn’t exported.

**Comment by Overload119** [comment]
Some things I've tried,

```
cd ~/Library/Application\ Support/Sublime\ Text\ 3/Installed\ Packages/
rm Package\ Control.sublime-package
cd ../Packages/
mkdir Package\ Control
cd Package\ Control/
git clone https://github.com/wbond/package_control.git .
git checkout osx-trust-list
```

Open Sublime, checkout to Master.
Reinstall some packages.

**Comment by wbond** [comment]
@Overload119 That branch will be a dead end. It has been merged into master and release as 3.2.1.
